rules/xcodeproj.bzl

```diff
@@ -334,12 +334,31 @@ def _xcodeproj_impl(ctx):
             "optional": True,
             "compilerFlags": "-fno-objc-arc",
         } for s in target_info.non_arc_srcs.to_list()]
-        asset_sources = [{
-            "path": paths.join(src_dot_dots, s.short_path),
-            "group": paths.dirname(s.short_path),
-            "optional": True,
-            "buildPhase": "none",
-        } for s in target_info.asset_srcs.to_list()]
+
+        asset_sources = []
+        for s in target_info.asset_srcs.to_list():
```

Comment:
can this logic be extracted into a function?

Reply:
> Cool! Can you add a screenshot to the descritption showing the before/after of how the Xcode file navigator shows the data models?

Yup will do this after I create test cases for this